specifyweb/frontend/js_src/lib/components/Forms/DeleteButton.tsx

```diff
@@ -56,6 +57,7 @@ export function DeleteButton<SCHEMA extends AnySchema>({
   deferred: initialDeferred = false,
   component: ButtonComponent = Button.Gray,
   onDeleted: handleDeleted,
+  focusedName: focusName,
```

Comment:
This works, but the solution is way too specific to this case.
Consider these things:
- DeleteButton does not know anything about the concept of "focused node" - and it shouldn't. At very least, rename the prop
- Adding a formatted node name would be great not just in the tree, but in all the places were DeleteButton is used - since you have access to the resource, you can call `format()` function to get a formatted string (see usages)